### ISSUE TITLE
README update

### ISSUE THREAD

**Issue by Sgtpluck** [thollander/actions-comment-pull-request#23]
It is great to see that you've moved to the `main` branch terminology! 

Can you please update your README to reflect that? Thank you so much!

```
- name: Comment PR
  uses: thollander/actions-comment-pull-request@main
```

**Comment by cbouwense-ikea** [comment]
I noticed this was causing an issue for my team as well, but simply changing from @master to @main didn't immediately solve our problem. Wondering if doing this in your own workflow didn't cause any issues? @Sgtpluck If it was fine for you then we probably have an issue of our own!

**Comment by Sgtpluck** [comment]
@cbouwense-ikea it did seem to solve it for our team! (I'll update if we see any problems going forward, though.)

**Comment by cbouwense-ikea** [comment]
Okay thanks!

**Comment by cbouwense-ikea** [comment]
@Sgtpluck bit of a newbie question: for a workflow update to be picked up, does it have to reside on your `default` branch?

**Comment by Sgtpluck** [comment]
@cbouwense-ikea it's picked up on whatever branch the change is made on!

so if you have it on a local branch, the update should be present for the action on that branch -- it won't be present on any other branch.

if you've merged it into your `default` branch but you have a local branch that hasn't had new code rebased or merged in, it won't automatically pick up the update. you'll have to rebase or merge the `default` code into that branch.

let me know if that doesn't make sense!

**Comment by cbouwense-ikea** [comment]
Hmm okay, that's the way I hoped it worked, but it doesn't seem to be so for me! I have a branch to change the `master` to `main`, it has the following change
```
-       uses: thollander/actions-comment-pull-request@master
+       uses: thollander/actions-comment-pull-request@main
```
and when I push that change and my action runs again, it fails with 
```
Unable to resolve action `thollander/actions-comment-pull-request@master`, unable to find version `master`
```
even though, when I take a look at the workflow file it's supposedly using, it seems to have the proper change!
```
        uses: thollander/actions-comment-pull-request@main
```
Really scratching my head here. New to GitHub actions though so I'm assuming there's a simple explanation haha

**Comment by Sgtpluck** [comment]
@cbouwense-ikea that does sound puzzling! does your setup have any kind of caching going on? or could there be some reason it's trying to use your default branch's action? 

if you wanted to post a little more of your setup, i could try to help!

**Comment by cbouwense-ikea** [comment]
@Sgtpluck Think I found it: looks like our workflow uses `pull_request_target` which uses the base branch workflow files, which would explain why this isn't using the workflow file in the fix branch. 🎉

**Comment by Sgtpluck** [comment]
Nice catch!

**Comment by thollander** [comment]
Closed by #26
